arm-network/2017-03-01/swagger/virtualNetwork.json

```diff
@@ -1047,24 +1048,29 @@
         "currentValue": {
           "type": "number",
           "format": "double",
-          "description": "Current usage value."
+          "readOnly": true,
+          "description": "Indicates number of IPs used from the Subnet."
```

Comment:
Not likely, but the spec file that those changes are based upon specifies both `currentValue` and `limit` as doubles.